authors/apps/articles/tests/test_create_article.py

```diff
@@ -0,0 +1,183 @@
+        self.client = APIClient()
+        self.login_url = reverse('authentication:auth-login')
+        self.signup_url = reverse('authentication:auth-register')
+        self.article_listcreate = reverse('articles:articles-listcreate')
+        self.signup_data = {
+            "user": {
+                "username": "kennyg",
+                "email": "ken@me.com",
+                "password": "Kennyisme1!"
+                }}
+        self.signup_data2 = {
+            "user": {
+                "username": "kwammea",
+                "email": "kwame@me.com",
+                "password": "Kwameisme1!"
+                }}
+        self.create_article_data = {
+            "title": "A new story",
+            "body": "This is my story",
+            "description": "Here is my story",
+            "images": None
+        }
+        self.create_article_data2 = {
+            "title": "The beginning",
+            "body": "This begins now",
+            "description": "Here is my story",
+            "images": None
+        }
+        self.delete_article = {
+            "is_deleted": False
+        }
+
+    def signup_user_one(self):
+        """Function to register user one and return their token"""
+        register = self.client.post(self.signup_url,
+                                    self.signup_data,
+                                    format='json')
+        token = register.data['token']
+        return token
+
+    def signup_user_two(self):
+        """Function to register user two and return their token"""
+        register = self.client.post(self.signup_url,
+                                    self.signup_data2,
+                                    format='json')
+        token = register.data['token']
+        return token
+
+    def test_fetch_all_articles(self):
+        """Tests to fetch all articles"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        del self.create_article_data2['images']
+        self.client.post(self.article_listcreate,
+                         self.create_article_data2,
+                         format='json',
+                         HTTP_AUTHORIZATION='token {}'.format(token))
+        response = self.client.get(self.article_listcreate, format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertIn("A new story", response.data[0]['title'])
+        self.assertIn("The beginning", response.data[1]['title'])
+        self.assertIn("This is my story", response.data[0]['body'])
+        self.assertIn("This begins now", response.data[1]['body'])
+        self.assertIn("Here is my story", response.data[0]['description'])
+        self.assertIn("Here is my story", response.data[1]['description'])
```

Comment:
It's difficult to understand where these strings `"Here is my story"`, etc are coming from. Please use the string directly from the object, like `self.create_article_data.description`

Reply:
done